### PR TITLE
Issue #9460: update example of AST for TokenTypes.LCURLY

### PR DESCRIPTION
**Pull request by Hreshi**
Resolves : Issue #9460: update example of AST for TokenTypes.LCURLY

site

### CODE REVIEW

**Comment by romani** [comment]
Please share details of ast print

**Comment by Hreshi** [comment]
AST print result
ast

Javadoc site according to latest commit

site

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 1]
```diff
@@ -1967,6 +1967,29 @@
     /**
      * A left curly brace (<code>{</code>).
      *
+     * <p>For example:</p>
+     * <pre>
+     * class App{
+     * int num
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:1973]: please add space before "{" and indent "int num" by 2 spaces.
This code is still for humans to read, formatting matters.

**Review by romani — CHANGES_REQUESTED** [review]
minor before merge:

**Comment by romani** [comment]
Please squash all commits in one. In PR should be single commit

**Comment by Hreshi** [comment]
Yes i am working on it

**Pull request closed without merging**